grammar.js

```diff
@@ -1282,11 +1280,7 @@ module.exports = grammar({
     // This has to be above _lex_low_id in order to take priority. Somehow precedences don't work
     // here...
     _lex_address: $ => token(seq(
```

Comment:
What about `_lex_b58` and make the prefix a field? So something like

```
_lex_b58: $ => /[0-9a-zA-Z]+/
b58: $ => seq(field("prefix", choice('ak', 'ok', ...), '_', field("data", $._lex_b58))
```

This would make it convenient to retrieve the prefix eg in ts queries

Reply:
I have just tried this. This has to be parsed as a single token using `token`, and it's not possible to use fields inside `token`.